src/models/validators/TestResultsSchemaCarSubmitted.ts

```diff
@@ -15,6 +15,11 @@ export const carSubmitted =
             .items(defectsCommonSchemaSpecialistTestsSubmitted)
             .optional(),
         }),
+          testTypesCommonSchemaSpecialistTestsSubmitted.keys({
```

Comment:
Even though they are required, shouldn't they allow null? similar to the logic for defects on annual tests?